go.mod

```diff
@@ -58,10 +58,12 @@ require (
 	github.com/jackc/pgtype v1.14.3
 	github.com/jackc/pgx/v4 v4.18.2
 	github.com/jackc/pgx/v5 v5.5.5
+	github.com/jeroenrinzema/psql-wire v0.11.1
 	github.com/jmoiron/sqlx v1.3.5
 	github.com/joho/godotenv v1.5.1
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/lensesio/tableprinter v0.0.0-20201125135848-89e81fc956e7
+	github.com/lib/pq v1.10.9
```

Comment:
The `psql-wire` has a dependency on `lib/pq` for setting OIDs. I think this needs fixing in that library before we can remove it here : https://github.com/jeroenrinzema/psql-wire/issues/39
I would like to keep the use of `psql-wire` since it provides lot of convenience over `pgx. I actually started with pgx directly but realised that I need to write lot of code which is already present in `psql-wire`. 
I can try fixing the original `psql-wire` issue.

Reply:
Okay, let's not worry about that now